Gemfile

```diff
@@ -1,5 +1,7 @@
 source "http://rubygems.org"
 
+ruby ">= 2.2"
```

Comment:
Should use ' instead of ".  Same on this [other spot](https://github.com/bespokepost/questionable/pull/22/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R1)